src/core/etl/tests/Flow/ETL/Tests/FlowTestCase.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Flow\ETL\Tests;
+
+use function Flow\ETL\DSL\rows;
+use Flow\ETL\{Config, Extractor, FlowContext, Rows};
+use PHPUnit\Framework\TestCase;
+
+abstract class FlowTestCase extends TestCase
+{
+    final public static function assertExtractedBatchesCount(
+        int $expectedCount,
+        Extractor $extractor,
+        ?FlowContext $flowContext = null,
+        string $message = '',
+    ) : void {
+        $flowContext = $flowContext ?? new FlowContext(Config::default());
```

Comment:
could you please use DSL functions in test suite instead of objects? 

```php
$flowContext = $flowContext ?? flow_context();
```

Reply:
Fixed in https://github.com/flow-php/flow/pull/1291/commits/cea9939f2fe2388a04cdf38687d18f77f7552da2